mailpoet/tests/integration/Models/SubscriberSegmentTest.php

```diff
@@ -15,6 +16,7 @@ class SubscriberSegmentTest extends \MailPoetTest {
 
   public function _before() {
     parent::_before();
+    ORM::raw_execute('SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO"');
```

Comment:
I opted to use Paris code here as this is a test class for a Paris model that we will eventually remove.